build.sbt

```diff
@@ -155,7 +155,7 @@ lazy val V = new {
   val mavenBloop = bloop
   val mdoc = "2.1.3"
   val scalafmt = "2.4.2"
-  val munit = "0.4.3"
+  val munit = "0.6.0"
```

Comment:
We have released 0.7.1 this morning, which is unchanged except for the newly added ScalaCheck integration

Reply:
Ahhh you guys are moving too fast 😆